src/xdocs/idea.xml

```diff
@@ -54,6 +54,11 @@
           <img alt="screenshot" src="images/gui_screenshot_debug_idea.png"/>
         </span>
         <br/><br/>
+        Open the Test's source file by double-click on it in a source tree as is shown:<br/><br/>
+        <span class="wrapper block">
+          <img alt="screenshot" src="images/gui_screenshot_click_Test_File.png"/>
```

Comment:
The new image looks very skewed compared to the others. It's like it was resized but not kept proportional to it's original. Compared to other images, left hand side looks a bit blurry and hard to read.

Can we please fix this?

Reply:
Sure I'd do that.

Reply:
Done!

Reply:
Please leave resolving items to reviewers.